server/server_test.go

```diff
@@ -16,7 +21,7 @@ func TestNewAdminServer(t *testing.T) {
 	}
 	server := newAdminServer(cfg, http.HandlerFunc(handler))
 	if server.Addr != "prebid.com:6060" {
-		t.Errorf("Admin server address should be %s. Got %s", "prebid.com:6060", server.Addr)
+		t.Errorf("Admin server address should be %s. Got %s", "prebid.com:6060\n", server.Addr)
```

Comment:
Super nitpick: can you please remove new lines at the end of error logs? There are couple of them.